flutter_local_notifications_linux/lib/src/platform_info.dart

```diff
@@ -27,8 +27,15 @@ class LinuxPlatformInfo {
         final int pid = _posix.getpid();
         final int userId = _posix.getuid();
         final int sessionId = _posix.getsid(pid);
+        final Map<String, String> env = Platform.environment;
+        final String? tmpdir = env['TMPDIR'];
         runtimeDir = Directory(
```

Comment:
@proninyaroslav sorry I missed this before but given this change has been made, this would require the API docs for `runtimePath` to be updated as well right?

Reply:
Yes, thank you.